content/post/094-gossipsub-v1.1.md

```diff
@@ -0,0 +1,80 @@
+---
+date: 2020-05-19
+url: 2020-05-19-gossipsub-v1.1
+title: Gossipsub v1.1 brings new hardening extensions to libp2p PubSub 
+author: David Dias
+header_image: 094-gossipsub-v1.1-headerimage.jpeg
+tags: ipfs, libp2p, pubsub, gossipsub
+---
+
+<p align="center">
+  <img width="60%" src="/header_images/094-gossipsub-v1.1-headerimage.jpeg">
+</p>
+
+The gossip you’ve heard on the streets is correct, Gossipsub v1.1 is here and it packs several security hardening extensions - along with new testing, documentation, and an updated specification.
+
+The Gossipsub Task Force has been hard at work exploring and analyzing various attack vectors on public and permissionless messaging networks. With that knowledge, we’ve crafted and iterated on mitigating strategies to make our beloved libp2p PubSub Router work in adversarial environments. The result is Gossipsub v1.1 🚀
+
+## 🔍 What is Gossipsub v1.1
+
+The main focus for Gossipsub v1.1 is **security**. In addition to the speedy message propagation guarantees from v1.0, this new version features several hardening extensions that make Gossipsub more resilient to a wide range of attacks. Some highlights include:
+
+1) [Peer scoring](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#peer-scoring): peers now monitor their directly connected neighbours using a score function that reflects the peer’s usefulness. Well-behaving peers are kept in the mesh, while badly-behaving ones are dropped in order to protect the network from malicious actors.
+
+2) [Adaptive gossip dissemination](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#adaptive-gossip-dissemination): In this new version, you can adjust the amount of peers in which your node gossips to with the gossip factor. This enables you to increase/decrease the amount of gossip, while keeping a baseline.
+
+3) [Opportunistic Grafting](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#opportunistic-grafting): A new mechanism used to cherry pick well-behaving peers that are currently not part of a node’s mesh. Those well-behaving peers are then opportunistically grafted to improve the mesh’s health in terms of median score of participating peers. 
```

Comment:
```suggestion
3) [Opportunistic Grafting](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#opportunistic-grafting): Grafting is a new mechanism used to cherry pick well-behaving peers that are currently not part of a node’s mesh. Those well-behaving peers are then opportunistically inserted into the mesh to improve the median score of participating peers. 
```
Shouldn't use "grafting" to explain what grafting is. Assume it's inserting them into the mesh? maybe "connected" is a better word?